robyn/__init__.py

```diff
@@ -239,6 +242,7 @@ def _add_openapi_routes(self, auth_required: bool = False):
             is_const=True,
             auth_required=auth_required,
         )
+        self.exclude_response_headers(["/docs"])
```

Comment:
what do you think about `/openapi.json` ?

Reply:
should I add it? I didn't think of a case that should be added ...

- update: just discovered all of the types! ref: https://stackoverflow.com/a/48704300/9652621, will add!